main/src/ca/uwaterloo/flix/language/phase/Lowering.scala

```diff
@@ -573,6 +573,9 @@ object Lowering {
         val t = visitType(tpe)
         val parExp = mkParApply(Expression.Apply(e, es, t, pur, eff, loc1))
         Expression.Cast(parExp, None, Some(Type.Pure), Some(Type.Empty), t, pur, eff, loc0)
+
+      case _ =>
```

Comment:
We don't use wildcards, because then we don't get warnings when we change the AST. We should probably add that to STYLE.md